gitversion.yml

```diff
@@ -1,33 +1,45 @@
 assembly-versioning-scheme: MajorMinorPatch
-mode: ContinuousDeployment
+mode: Mainline
 next-version: 1.29.0
-continuous-delivery-fallback-tag: ""
+
 branches:
   master:
+    mode: ContinuousDeployment
     regex: master
     tag: dev
-    increment: none
-  beta:
-    regex: release/beta/
-    tag: 'beta'
-    increment: none
-    source-branches: ['master']
+    increment: Minor
+
   stable:
-    regex: release/stable/
+    regex: release/stable/.*
     tag: ''
-    increment: none
+    increment: Patch
     source-branches: ['master']
+    is-mainline: true
+
+  beta:
+    mode: ContinuousDeployment
+    regex: release/beta/.*
+    tag: 'beta'
+    increment: inherit
+    source-branches: ['master']
+
   dev:
     regex: dev/.*?/(.*?)
     tag: dev.{BranchName}
-    source-branches: ['master']
+    source-branches: ['master', 'release', 'projects', 'feature']
+    increment: inherit
+
   projects:
```

Comment:
Shouldn't `project` and `feature` branches be mentioned in the doc?

Reply:
Indeed, will update.